src/app/modules/main/wallet_section/saved_addresses/controller.nim

```diff
@@ -29,8 +29,17 @@ proc init*(self: Controller) =
 proc getSavedAddresses*(self: Controller): seq[saved_address_service.SavedAddressDto] =
   return self.savedAddressService.getSavedAddresses()
 
+proc getFavouriteAddresses*(self: Controller): seq[saved_address_service.FavouriteDto] =
+  return self.savedAddressService.getFavouriteAddresses()
+
 proc createOrUpdateSavedAddress*(self: Controller, name, address: string): string =
   return self.savedAddressService.createOrUpdateSavedAddress(name, address)
 
 proc deleteSavedAddress*(self: Controller, address: string): string =
   return self.savedAddressService.deleteSavedAddress(address)
+
+proc setFavouriteAddressState*(self: Controller, address: string, favourite: bool): string =
+  if favourite:
+    return self.savedAddressService.addToFavourite(address)
+  else:
+    return self.savedAddressService.deleteFavouriteAddress(address)
```

Comment:
I use VSCode and QtCreator and never checked if it is added by default but I usually try to keep/restore it if appending anything just because it is there and for the sake of the diff.

In practice, I have never had issues with end of the line even when working with the old C++ standard but judging by this post the compiling with C++11 doesn't generate issues anymore: https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning